app/models/concerns/spotlight/resources/open_graph.rb

```diff
@@ -23,10 +23,6 @@ def opengraph_properties
           ["#{k.parameterize('_')}_tesim", v]
         end]
       end
-
-      def to_solr
```

Comment:
I see these properties being used in our tests but otherwise we don't seem to be indexing `opengraph_properties`.